server/middleware/setUpCurrentUser.ts

```diff
@@ -11,7 +11,7 @@ export default function setUpCurrentUser(services: Services): Router {
   const router = Router({ mergeParams: true })
 
   router.use(auth.authenticationMiddleware(tokenVerifier))
-  router.use(populateCurrentUser(services.userService))
+  router.use(populateCurrentUser(services.userService, services.expectedArrivalsService))
```

Comment:
umm.. this seems a bit off - populateCurrentUser is domain agnostic and we're pushing in the thing responsible for a chunk of our domain.

The only reason we need this is for: 
`expectedArrivalsServices.getPrison(user.activeCaseLoadId)`

the concept of `getPrison` doesn't seem very related to conceptual area of expected arrivals.

Maybe we should extract it into its own service

Reply:
Sorted